### ISSUE TITLE
Better error message for packages not present in old releases

### ISSUE THREAD

**Issue by LTLA** [Bioconductor/BiocManager#85]
Over the past few weeks, I've gotten queries about the inability to install packages that are only present in 3.12 (e.g., **scuttle**, **celldex**) from people who are still running on 3.11. The current failure message looks like:

```
> BiocManager::install("celldex")
Bioconductor version 3.11 (BiocManager 1.30.10), R 4.0.1 (2020-06-06)
Installing package(s) 'celldex'
Warning message:
package ‘celldex’ is not available (for R version 4.0.1)
```

Which is not really true; it is available on this version of R, just on a higher version of Bioconductor. I could avoid a lot of questions if `install()` gave a more informative message, possibly something like:

```
package 'celldex' is not available in Bioconductor version 3.11. However,
it is present in the latest release of Bioconductor (version 3.12). Use 
'install(version="3.12")' to upgrade your current version of Bioconductor.
```

**Comment by LTLA** [comment]
Since this issue was raised, I can count approximately 10 incidences of people coming to me with this error. Who knows how many more people just gave up without contacting me.

**Comment by mtmorgan** [comment]
it is too expensive / complicated to find what versions of r / bioc is required for a package, but maybe there is scope for a function `available_versions()` that would do that / the user could be pointed toward in an updated error message.  pull request?

**Comment by hpages** [comment]
But at least you could say something like "package 'celldex' is not available in Bioconductor version 3.11". This should be relatively cheap/simple, and would already be an improvement over "package ‘celldex’ is not available for R version 4.0.1", which is not true.

**Comment by LiNk-NY** [comment]
I agree with Martin, looking across versions is quite expensive and complicated. I've added the 'cheap/simple' fix that Hervé mentioned. Thanks!
